smart_proxy_tests.sh

```diff
@@ -25,7 +25,7 @@ function install_reqs() {
 function prepare_venv() {
     echo "Preparing environment"
     # shellcheck disable=SC1091
-    virtualenv -p python3 venv && source venv/bin/activate && pip install -r requirements/insights_results_aggregator_mock.txt || exit 1
+    virtualenv -p python3 venv && source venv/bin/activate && install_reqs || exit 1
```

Comment:
some funcionts use `install_reqs` and others don't. Can we standardize?

Reply:
this should be solved